hvplot/converter.py

```diff
@@ -387,6 +396,22 @@ def __init__(self, data, x, y, kind=None, by=None, use_index=True,
             param.main.warning('Plotting {kind} plot with parameters x: {x}, '
                                'y: {y}, by: {by}, groupby: {groupby}'.format(**kwds))
 
+    def _process_symmetric(self, clim, symmetric, kwds):
+        if clim is None and symmetric is None:
+            if is_xarray(self.data):
+                data = self.data[list(self.data.data_vars)[0]]
+            else:
+                data = self.data[kwds['C']]
+
+            cmin = np.nanquantile(data, 0.05)
+            cmax = np.nanquantile(data, 0.95)
+            clim = (cmin, cmax)
+            divergent = cmin < 0 and cmax > 0
+            if divergent:
+                symmetric = True
+            # TODO: add cmap from hv.config
```

Comment:
Note that #258 adds a default diverging colormap so that would be the right thing to use here.

Reply:
Thanks! Not sure if the way I implemented it is logical though. symmetric is under plot_opts, but the cmaps are decided under process_style which uses style_opts